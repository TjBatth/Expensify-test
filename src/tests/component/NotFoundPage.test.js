import React from 'react'
//import ReactShallowRenderer from 'react-test-renderer/shallow'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json';
import NotFoundPage from '../../component/NotFoundPage'

test('should NotFoundPage renderer successfully',()=>{
    const wrapper=shallow(<NotFoundPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
})
