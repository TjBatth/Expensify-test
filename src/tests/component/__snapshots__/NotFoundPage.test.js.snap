// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should NotFoundPage renderer successfully 1`] = `
<div>
  Page is Not here !! 404 
  <Link
    replace={false}
    to="/"
  >
    Go Home 
  </Link>
</div>
`;
